spec/requests/inventories_spec.rb

```diff
@@ -26,7 +26,7 @@
     end
 
     it 'should have the exact page content' do
-      expect(response.body).to include('This page would be for a specific inventory')
+      expect(response.body).to include('Your Inventories')
```

Comment:
- [ ] Kindly check all the specs by running the following command. `bundle exec rspec spec`. 
Currently some some specs are failing related with the inventories
![failing-spec](https://user-images.githubusercontent.com/44057863/169500825-2c580e5d-c06e-4959-9a9d-24476fbd67b4.png)
.